src/game.cpp

```diff
@@ -5648,6 +5648,11 @@ void game::examine( const tripoint &examp )
             Pickup::pick_up( examp, 0 );
         }
     }
+    if( is_dangerous_tile( examp ) ) {
+        if( prompt_dangerous_tile( examp ) ) {
```

Comment:
Could be merged into one `if( is_dangerous_tile( examp ) && prompt_dangerous_tile( examp ) )` check.

Reply:
Good catch.